src/app/app.component.ts

```diff
@@ -66,6 +61,13 @@ export class AppComponent implements AfterViewInit, OnInit {
     // always set up during the initial page load, regardless of whether the first
     // page the donor lands on makes wider use of DonationService or not.
     this.donationService.deriveDefaultCountry();
+
+    if (window.location.host === 'donate.biggive.org') {
+      // donations are currently not working on the new domain. No-one should be visiting that domain yet, but
+      // in case they do we redirect them to the old domain for now:
+
+      window.location.href = "https://biggive.org.uk/" + window.location.pathname;
```

Comment:
@bdsl doesn't this need a "the"?

Reply:
It should be       `window.location.href = "https://donate.thebiggive.org.uk/" + window.location.pathname;`

Reply:
https://github.com/thebiggive/donate-frontend/pull/908
https://github.com/thebiggive/donate-frontend/pull/909